src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listItem = document.querySelectorAll('li');
```

Comment:
Ensure that the DOM is fully loaded before querying elements. You might want to wrap your code in a `DOMContentLoaded` event listener to ensure that `listItem` is populated correctly.